### ISSUE TITLE
UI is not started if makefile.buildLog is provided

### ISSUE THREAD

**Issue by cetygamer** [microsoft/vscode-makefile-tools#266]
If `makefile.buildLog` option is provided, extension does not activate UI (icon in the side bar) and does not register most commands.

Only commands available in such case:

![image](https://user-images.githubusercontent.com/14858753/149390281-58f5eff7-db61-419e-8314-0480f1360f6b.png)

Debug extension log (I stripped IntelliSense configuration and parsing of make output):

```
Configurations cached at /home/frim/editline/.vscode/configurationCache.log
No path to the makefile is defined in the settings file.
No folder path to the makefile is defined in the settings file.
Build log defined at "/home/frim/editline/.vscode/build.log"
Always pre-configure: false
Dry-run switches: --always-make,--keep-going,--print-directory
Additional compiler names: 
Exclude compiler names: 
No current configuration is defined in the workspace state. Assuming 'Default'.
No target defined in the workspace state. Assuming 'Default'.
No current launch configuration is set in the workspace state.
Default launch configuration: MIMode = undefined,
                    miDebuggerPath = undefined,
                    stopAtEntry = undefined,
                    symbolSearchPath = undefined
Configure on open: true
Configure on edit: false
Configure after command: false
Only .PHONY targets: false
Save before build or configure: true
Build before launch: true
Clear output before build: true
Ignore directory commands: true
compile_commands.json path: null
Deduced command 'make ' for configuration "Default"
Saving opened files before build.
Configuring from cache: /home/frim/editline/.vscode/configurationCache.log
Load configuration from cache elapsed time: 0.28
<STRIPPED>
Preprocessing: "/home/frim/editline/.vscode/build.log"
Preprocess elapsed time: 0
Parsing for IntelliSense.
Updating the CppTools IntelliSense Configuration Provider.
Parsing dry-run output for CppTools Custom Configuration Provider.
<STRIPPED>
No launch configurations have been detected.
Complete list of launch targets: 
Parsing for launch targets elapsed time: 0.013
Generating parse content for build targets.
Generating targets information with command: 'make all --print-data-base --no-builtin-variables --no-builtin-rules --question'
Writing the dry-run output: /home/frim/editline/.vscode/targets.log
Spawning child process with:
 process name: make
 process args: all,--print-data-base,--no-builtin-variables,--no-builtin-rules,--question
 working directory: /home/frim/editline
 shell type: default
Generating dry-run elapsed time: 0.019
Parsing for build targets from: "/home/frim/editline/.vscode/targets.log"
Found the following 134 build targets defined in the makefile: <STRIPPED>
Complete list of build targets: <STRIPPED>
Parsing build targets elapsed time: 0.008
Configure finished. The status for all the subphases that ran:
loadFromCache: return code = 0, elapsed time = 0.28
generateParseContent: return code = 0, elapsed time = 0
preprocessParseContent: return code = 0, elapsed time = 0
parseIntelliSense: return code = 0, elapsed time = 0.054
parseLaunch: return code = 0, elapsed time = 0.013
dryrunTargets: return code = 1, elapsed time = 0.019
parseTargets: return code = 0, elapsed time = 0.008
Configure succeeded.
Sending telemetry: eventName = configure
properties: firstTime = "false",makeDryRun = "false",readCache = "true",isClean = "true",processTargetsSeparately = "false",resetBuildTarget = "false",triggeredBy = "configure dirty (on open), settings (configureOnOpen)",buildTarget = "(unset)"
measures: numberBuildTargets = "134",numberLaunchTargets = "0",numberIndexedSourceFiles = "3",numberMakefileConfigurations = "0",totalElapsedTime = "0.429",exitCode = "0",loadFromCache.exitCode = "0",loadFromCache.elapsed = "0.28",generateParseContent.exitCode = "0",generateParseContent.elapsed = "0",preprocessParseContent.exitCode = "0",preprocessParseContent.elapsed = "0",parseIntelliSense.exitCode = "0",parseIntelliSense.elapsed = "0.054",parseLaunch.exitCode = "0",parseLaunch.elapsed = "0.013",dryrunTargets.exitCode = "1",dryrunTargets.elapsed = "0.019",parseTargets.exitCode = "0",parseTargets.elapsed = "0.008"
Configure elapsed time: 0.429
Sending telemetry: eventName = settings
properties: makefile.loggingLevel = "Debug",makefile.makeDirectory = "undefined",makefile.makefilePath = "undefined",makefile.buildLog = "...",makefile.extensionLog = "...",makefile.configureOnEdit = "false",makefile.configureAfterCommand = "false"
```

**Comment by andreeis** [comment]
@cetygamer, yes we found this after 1.3.1 and it will be fixed with 1.4.0 (soon).

https://github.com/microsoft/vscode-makefile-tools/pull/256

In the meantime, you can manually run "Makefile: Configure" from the VSCode commands pallette or you can install manually a vsix (downloaded [from here](https://github.com/microsoft/vscode-makefile-tools/actions/runs/1699803053)) which contains the fix. Use "Install from vsix" context menu of "..." in the extensions left side tab.

**Comment by andreeis** [comment]
@cetygamer, we released Makefile Tools 0.4.0 today which contains the fix for this issue and also worth noting, localization support. If you happen to use VSCode language packs, give Makefile Tools a try in your preferred language.
Let us know if you encounter any issues: with the translated strings, any other bugs or if you have any feedback for the future of this extension.
